### PR TITLE
Secondary sales

### PR DESCRIPTION
**Pull request by vince0656**

### CODE REVIEW

File: contracts/token/TwistedSisterToken.sol [hunk 1]
```diff
@@ -41,11 +43,16 @@ contract TwistedSisterToken is CustomERC721Full, ITwistedSisterTokenCreator {
...
+    constructor (
+        string memory _tokenBaseURI,
+        ITwistedSisterAccessControls _accessControls,
+        uint256 _transfersEnabledFrom,
+        TwistedSisterArtistFundSplitter _artistFundSplitter
+    ) public CustomERC721Full("twistedsister.io", "TWIST") {
         accessControls = _accessControls;
         tokenBaseURI = _tokenBaseURI;
         transfersEnabledFrom = _transfersEnabledFrom;
+        artistFundSplitter = _artistFundSplitter;
```
Comment (jamesmorgan) [inline contracts/token/TwistedSisterToken.sol]: Add a mutator so we can switch out if needed

Comment (andygray) [reply contracts/token/TwistedSisterToken.sol]: yeah. good idea.

File: contracts/token/TwistedSisterToken.sol [hunk 2]
```diff
@@ -92,9 +99,37 @@ contract TwistedSisterToken is CustomERC721Full, ITwistedSisterTokenCreator {
...
+            // 70% seller
+            uint256 sellersSplit = singleUnitOfValue.mul(70);
+            (bool fromSuccess, ) = from.call.value(sellersSplit)("");
+            require(fromSuccess, "Failed to send funds to the seller");
+        }
+    }
+
+    function _sendValueToTokenHolders(uint256 _value) private {
+        uint256 individualTokenHolderSplit = _value.div(tokenIdPointer);
+        for(uint i = 1; i <= tokenIdPointer; i++) {
```
[the remark this answers is not in the document]
Comment (andygray) [reply contracts/token/TwistedSisterToken.sol]: I would just use `uint256` everywhere for this project

File: test/token/twisted/ERC721Full.test.js [hunk 3]
```diff
@@ -267,6 +294,52 @@ contract('ERC721 Full Test Suite for TwistedToken', function ([
...
+                const anotherBalance = await balance.tracker(another);
+                const toBalance = await balance.tracker(to);
+                const toAnotherBalance = await balance.tracker(toAnother);
+
+                // approve the new owner to send the value and make the tx
+                const tokenOneId = 1;
+                ({receipt: this.receipt} = await this.token.approve(to, tokenOneId, {from: owner}));
+                const approvalGasSpent = gasSpent(this.receipt);
+
+                ({receipt: this.receipt} = await this.token.transferFrom(owner, to, tokenOneId, {from: to, value: oneEth}));
```
[the remark this answers is not in the document]
Comment (andygray) [reply test/token/twisted/ERC721Full.test.js]: err. Will test on Rinkeby.

**Review by andygray — APPROVED** [review]

**Pull request merged**